Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,92 @@ Meaning of columns:
+| `IfcCartesianPointList3D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCartesianTransformationOperator2D`| :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCartesianTransformationOperator3D`| :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCartesianTransformationOperator2DnonUniform` | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCircle`                           | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCircularArcSegment2D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcClothoid`                         | :x:                  | `CurveConverter`           | -                     | [#304](https://github.com/tumcms/Open-Infra-Platform/issues/304) |
+| `IfcCompositeCurve`                   | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCompositeCurveOnSurface`          | :x:                  | -                          | -                     | -     |
+| `IfcCompositeCurveSegment`            | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcCsgSolid`                         | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcCurveSegment`                     | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcDirection`                        | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcEllipse`                          | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcExtrudedAreaSolid`                | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcExtrudedAreaSolidTapered`         | :x:                  | -                          | -                     | -     |
+| `IfcFaceBasedSurfaceModel`            | :heavy_check_mark:   | `FaceConverter`            | -                     | -     |
+| `IfcFacetedBrep`                      | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcFacetedBrepWithVoids`             | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcFillAreaStyleHatching`            | :x:                  | -                          | -                     | -     |
+| `IfcFillAreaStyleTiles`               | :x:                  | -                          | -                     | -     |
+| `IfcFixedReferenceSweptAreaSolid`     | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcGeometricCurveSet`                | :x:                  | -                          | -                     | -     |
```

Comment:
This one's supported as well. Double-check, please.

Reply:
![image](https://user-images.githubusercontent.com/76201952/110946911-c66ad300-833f-11eb-8e18-73ae6ba390c6.png)
Hmm...

Reply:
See the last comment - we're done :)

Similarly to `IfcBooleanClippingResult`, it's only a Where rule, so we already support everything.